packages/backend/src/account/service.test.ts

```diff
@@ -22,10 +22,14 @@ describe('Account Service', (): void => {
   const mockMessageProducer = {
     send: jest.fn()
   }
+  const overrideConfig = {
```

Comment:
Could this instead happen in `createTestApp` like how it currently is in `main` (what I removed in the [testcontainers pr](https://github.com/coilhq/rafiki/pull/44/files#diff-189b2fcc08bab2590fbd9ef5e29200b7d7bbbe3fbf4f486fa9ae541f2e929da9L18-R97)) instead of each test file?

Reply:
I've moved it to the config file. It's needed when we create the IoC container, which happens before `createTestApp` is run.